scriptworker/test/test_task.py

```diff
@@ -342,39 +365,73 @@ def test_reclaim_task_non_409(context, successful_queue, event_loop):
         )
 
 
+@pytest.mark.parametrize("proc", (None, 1))
 @pytest.mark.asyncio
-async def test_reclaim_task_mock(context, mocker, event_loop):
+async def test_reclaim_task_mock(context, mocker, proc):
+
+    count = []
 
     async def fake_reclaim(*args, **kwargs):
         return {'credentials': context.credentials}
 
+    async def fake_kill_task(*args):
+        count.append(args)
+
     def die(*args):
         raise taskcluster.exceptions.TaskclusterRestFailure("foo", None, status_code=409)
 
+    context.proc = proc
     context.temp_queue = mock.MagicMock()
     context.temp_queue.reclaimTask = fake_reclaim
     mocker.patch.object(pprint, 'pformat', new=die)
+    mocker.patch.object(swtask, 'kill_task', new=fake_kill_task)
     await swtask.reclaim_task(context, context.task)
+    if proc:
+        assert len(count) == 1
+    else:
+        assert len(count) == 0
 
 
 # max_timeout {{{1
-def test_max_timeout_noop(context):
-    with mock.patch.object(swtask.log, 'debug') as p:
-        swtask.max_timeout(context, "invalid_proc", 0)
-        assert not p.called
+@pytest.mark.asyncio
+async def test_max_timeout_noop(context, mocker):
 
+    called = []
 
-def test_max_timeout(context, event_loop):
+    async def fake_kill(*args):
+        called.append(args)
+
+    mocker.patch.object(swtask, 'kill_task', new=fake_kill)
+    await swtask.max_timeout(context, "invalid_proc", 0)
+    assert called == []
+
+
+@pytest.mark.asyncio
+async def test_max_timeout_mock(context, mocker):
+
+
```

Comment:
It looks like some spurious blank lines got added to the beginning of a few of these methods.